transformer/transformer_test.go

```diff
@@ -309,8 +309,8 @@ func TestCustom(t *testing.T) {
 }
 
 func TestCustomFail(t *testing.T) {
-	r := &rpb.Request{Html: "<html ⚡><lemur>", Config: rpb.Request_CUSTOM, Transformers: []string{"does_not_exist"}}
-	if html, _, err := Process(r); err == nil {
+	r := rpb.Request{Html: "<html ⚡><lemur>", Config: rpb.Request_CUSTOM, Transformers: []string{"does_not_exist"}}
```

Comment:
@banaag Oops, this should have been changed in Google and then sync'd out. Can you send a CL?

Reply:
Sent.